source/common/config/cds_json.cc

```diff
@@ -102,7 +102,9 @@ void CdsJson::translateCluster(const Json::Object& json_cluster,
   const std::string name = json_cluster.getString("name");
   Utility::checkObjNameLength("Invalid cluster name", name);
   cluster.set_name(name);
-
+  if (json_cluster.hasObject("alt_stat_name")) {
```

Comment:
Do we need v1 support for this feature? If so, can you add v1 docs? I don't think they were added.

Reply:
I was not thinking that this is needed for v1. If you think it makes sense to have it there, I am ok to update the docs.

Reply:
Yeah, let's drop v1, so remove any changes to `cds_json.cc` and `config_schemas.cc` unless you feel there is a need for this.

Reply:
with out those some test cases are failing I guess. Instead of removing the tests, I have added this code. So better to leave it like that.

Reply:
Which tests? Seems that there is some bug if adding a new fields is causing v1 based tests to fail?

Reply:
oh.sorry. the test that I added is failing because it is constructing the cluster information from json object.

Reply:
Since we don't want v1 support for this, you'll need to make the test use a v2 config.  I'm guessing you're getting the error because the test uses a v1 config.

Reply:
@ggreenway yes, making that change.will update in few mins